src/components/ModalEditWater/ModalEditWater.jsx

```diff
@@ -7,29 +7,42 @@ import {
   StyledModalAddTracker,
   StyledModalAddValue,
   StyledModalForm,
+  TimeGlobalStyles,
 } from '../ModalAddWater/ModalAddWaterStyled.js';
 import {
+  ModalEditDateWrap,
   StyledModalEditInput,
   StyledModalEditStat,
 } from './ModalEditWater.styled.js';
 import { useState } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
 
 const ModalEditWater = () => {
   const [counter, setCounter] = useState(50);
+  const [time, setTime] = useState(new Date());
 
-  const MIN_VALUE = 0;
-  const MAX_VALUE = 1500;
+  const handleUpdate = (evt) => {
```

Comment:
you have absolutely the same function in the component [ModalAddWater.jsx] - please move it to separate helper file and reuse in both components, - it will ensure clean code without duplications

Reply:
I will. If we’re talking about clean code, please report wrong file names for styled components and color settings that have to be replaced with var(s) also. Those are very common so far, ty